app/main.py

```diff
@@ -202,14 +202,15 @@ def rabbit_status_callback(self, response):
             status = res.get('status')
             logger.info("Rabbit MQ health check response {}".format(status))
             if status == "ok":
-                logger.info('Setting status now')
+                logger.info('Rabbit MQ health ok')
```

Comment:
Worth logging whether health is good or bad and report it:
e.g.
```
logger.info("Checked rabbit health", health="ok")
```
or
```
logger.info("Checked rabbit health", health="bad")
```

Reply:
Should log overall health response in ``determine_health()`` too - or could just do it there with something like:
```
logger.info("Checked app health", app=False, rabbit=True, ftp=False)
```
Could probably have better names for the keys